Holding Facility Overhaul

Zytolg opened the pull request:
## About The Pull Request

When it comes to mapping this year, I believe we have done an absolutely fantastic job with bringing our stations into 2021. Everyone who has had a hand in that seriously deserves a pat on the back because you have all done amazing work. You have done so well, in fact that some of our older maps in comparison, feel seriously out of place. Some of these aren't a problem, yet. That's more due to a lack of use if anything though (I'm looking at you, Gateway Missions). Some of them, are engaged with by our players pretty often, and thus deserve some attention. This PR falls under the latter. 

## Why It's Good For The Game

Maps are the foremost thing that immerse our players in the game world. They are the metaphorical stage to /tgstations/ play. When maps look out of place, like they don't belong in the game anymore, it can sour the overall experience. The Holding Facility is a map that players engage with on a consistent enough basis to warrant a revisit. This updated version promises to be more fun to explore, while providing a tad bit more for enterprising players to do. 


![Holding Facility Done](https://user-images.githubusercontent.com/33048583/135587539-d13c50a0-a989-43cc-99d5-aae9a8722f5f.png)

## Changelog

:cl:
expansion: Remaps the Holding Facility into 2021 Standards
code: Indestructible paper wall support. It's better than the alternative. 
/:cl:

Zytolg committed:
Wraps it up, Folks

Zytolg committed:
Plasmeme Proofing

Mooshimi commented:
I'm excited to see this merged!

Maurukas commented:
This is incredibly good looking, and it should make one of the game's less pleasant mechanics more bearable. Now for the problems.

<details>
<summary>Active turfs here:</summary>
This was significant enough to depressurize the area.
![image](https://user-images.githubusercontent.com/66576896/135762452-fb4f8cde-c785-47de-bfcc-a76ea0df7f1b.png)

![image](https://user-images.githubusercontent.com/66576896/135762465-6081a97c-b48a-4f33-9a2d-825296c233b9.png)

</details>

<details>
<summary>These areas have little/zero lighting. The bathroom is probably not intentional:</summary>

![image](https://user-images.githubusercontent.com/66576896/135762550-1cc67b6b-7468-4525-884c-ec5f45f42890.png)

</details>

<details>
<summary> You've expanded the medical treatment options, provided cyberhearts, but no defib. </summary>

![image](https://user-images.githubusercontent.com/66576896/135762764-332592ca-d61b-4b09-8a2d-eb83bb1125ee.png)
</details>

Commit:
AT's, Lighting, Defibs

@Maurukas it's funny that the turfs I don't test are the ones that come back to bite me. Should be good now.
